codegen.yml

```diff
@@ -0,0 +1,16 @@
+overwrite: true
+schema: "https://crispy-api/graphql"
```

Comment:
is this url correct?
@Bazillius do we need nginx over `web`and `api` to proxy requests?

Reply:
This URL would be also changes when we will know exactly which is correct